Small fixes after guides review (#105)
kiegroup/kogito-docs: serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-openapi-based-services.adoc

+
Once you have access to the OpenAPI specification files, you can define the functions in the Serverless Workflow. 

> You can also use the link:{kogito_sw_examples_url}/serverless-workflow-temperature-conversion[serverless-workflow-temperature-conversion] example application as a reference.

Reviewer: For consistency, the example applications are written as code. 
```suggestion
You can also use the link:{kogito_sw_examples_url}/serverless-workflow-temperature-conversion[`serverless-workflow-temperature-conversion`] example application as a reference.
```

Author: ok...
@hmanwani-rh the main reason I wanted to change this is that in the documentation, with this formatting, we don't make the distinction between a example application mention and a link ...

- `link:{kogito_sw_examples_url}/serverless-workflow-temperature-conversion[`serverless-workflow-temperature-conversion`]`
- `serverless-workflow-temperature-conversion`

both appears as same formatting in the documentation and we may not know that we can click on it ...

Reviewer: +1 on the formatting. 
Maybe we can change the style after DP1 to show the code links with an underline. 

CC @ricardozanini

Author: do you need an issue for that ?

Reviewer: Sure, we can create one.

<1> Function definition for OpenAPI multiplication function
<2> Function definition for OpenAPI subtraction function

[NOTE]
====

> Note that in the above example, we are storing the `multiplication` and `substraction` OpenAPI files into the `src/main/resources/specs` folder of our application.

Reviewer: ```suggestion
Note that in the previous example, the `multiplication` and `subtraction` OpenAPI files are stored in the `src/main/resources/specs` folder of the application.
```
